### ISSUE TITLE
FS17 upgrade?

### ISSUE THREAD

**Issue by DeckerMMIV** [DeckerMMIV/FarmSim_Mod_SoilMod#96]
Your question: Is this going to be upgraded to FS17?

My answer: Only time will tell. - Need some time to actually play the vanilla game.

**Comment by mereman** [comment]
Take your time mate the one mod I really miss at the moment is Glance.

**Comment by Dzi4d3k** [comment]
Yes, it is still time, but with our team experience with SoilMod, vanilla FS17 still looks to arcade type game for us :(

**Comment by Paul2510** [comment]
HI! Will there be the Soil Mod for Ls 17 !! I missed a lot! The functions of ls 17 sid nict so bad but Soil Mod is around worlds better! LG

**Comment by Paul2510** [comment]
HI! Will there be the Soil Mod for Ls 17 !! I missed a lot! The functions of ls 17 sid nict so bad but Soil Mod is around worlds better! LG

**Comment by DeckerMMIV** [comment]
@Paul2510 - Only time will tell. I am currently doing some preparations and experimentation.

**Comment by Paul2510** [comment]
Many thanks for the answer! LG

**Comment by Dwinesk** [comment]
Hi. Years on and some of us have not stopped wishing for a version of this mod for FS17. Just imagining having soil mod, seasons, gearbox and more realistic all in the same game at the same time would be an unbelievable thing of beauty. I'm no computer whiz but I think that fs15 is quite close to fs17 so conversion should be possible??? So how about it?
